skimage/restoration/inpaint.py

```diff
@@ -156,6 +185,7 @@ def _inpaint_biharmonic_single_region(image, mask, out, neigh_coef_full,
     # set use_umfpack to False so float32 data is supported
     result = spsolve(matrix_unknown, rhs, use_umfpack=False,
                      permc_spec='MMD_ATA')
+    1 / 0
```

Comment:
I don't understand this addition. `1/0`

Reply:
I did that as a quick way to cause a failure for Ipython debug magic (poor man's breakpoint!) and just forgot to remove it! I was just double-checking that the `matrix_unknown` and `rhs` dimensions matched the description above.